src/reporting/ReportingWindow.sol

```diff
@@ -216,7 +226,13 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
     }
 
     function getRequiredReportsPerReporterForlimitedReporterMarkets() public afterInitialized constant returns (uint256) {
-        return getTargetReportsPerLimitedReporterMarket() * limitedReporterMarkets.count() / registrationToken.totalSupply();
+        uint256 _numLimitedReporterMarkets = limitedReporterMarkets.count();
+        uint256 _requiredReports = getTargetReportsPerLimitedReporterMarket() * _numLimitedReporterMarkets / registrationToken.totalSupply();
+        // We shouldn't require more reporting than is possible.
+        if (_requiredReports > _numLimitedReporterMarkets) {
+            return _numLimitedReporterMarkets;
+        }
+        return _requiredReports;
```

Comment:
Consider: This could be simplified with `return _requiredReports.min(_numLimitedReporterMarkets);`